addons/account/models/chart_template.py

```diff
@@ -218,6 +218,30 @@ def _create_demo_data(self):
             # Do not rollback installation of CoA if demo data failed
             _logger.exception('Error while loading accounting demo data')
 
+    def _get_extra_modules(self):
+        country_code = self.get_xml_id()[self.id].split('.')[0][5:]
+        extra_modules = [
+            module
+            for module, countries in {
+                'account_check_printing': ['US', 'CA'],
+                'account_edi_proxy_client': ['IT'],
+                'base_vat': [
+                    'AT', 'BE', 'CA', 'CO', 'DE_SKR03', 'DE_SKR04', 'EC', 'ES', 'ET', 'FR', 'GR',
+                    'IT', 'LU', 'MX', 'NL', 'NO', 'PL', 'PT', 'RO', 'SI', 'TR', 'GB', 'VE', 'VN',
+                    'SYSCOHADA',
+                ],
+            }.items()
+            if country_code.upper() in countries
```

Comment:
country_code is always upper

Reply:
except for de_skr03 maybe

Reply:
But why such nasty split as the previous code used the country code

Reply:
It is not the country code, it is the module name